AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts

```diff
@@ -89,6 +96,7 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
     originalNonWorkHours: EditVhoNonAvailabilityWorkHoursModel[];
     isEditing = false;
     isSaving = false;
+    isFiltered = false; //form is not filtered by default
```

Comment:
I don't think these comments are necessary, it's already quite clear what the code is doing.

Reply:
sorry @arellison I should have been clearer - I think all those comments can go. They don't really add very much

Reply:
I've removed all the comments now @oliver-scott

Reply:
great, just a linting issue to fix. `npm run fix-lint`, commit and push should do it :)

Reply:
I've ran npm run fix-lint, committed and pushed now @oliver-scott